manifests/init.pp

```diff
@@ -81,6 +84,22 @@
   #Merged values provides by user and default values
   $_real_padoptions = merge($default_padoptions, $padoptions)
 
+  #Install choosing plugins
+  $plugins_list.each |String $_pname, Variant[Boolean, Undef] $_penable| {
+  if $_penable == true {
+    contain "etherpad::plugins::${_pname}"
+    Class["etherpad::plugins::${_pname}"]
```

Comment:
This line isn't needed. `contain` already ensures the class is included

Reply:
Fixed.